src/renderer/containers/UploadSelectionPage/DragAndDropPrompt/index.tsx

```diff
@@ -29,24 +31,14 @@ export default function DragAndDropPrompt(props: DragAndDropPromptProps) {
         }
     };
 
-    let helpText: string | null = null;
-    if (props.uploadType) {
-      helpText = props.uploadType === UploadType.File
-        ? "Accepted file formats include: .czi, .tiff, .ome.tiff, .csv, ..."
-        : "Accepted file formats include: .zarr, .sldy, ..."
-    }
-
     return (
         <div className={styles.content}>
-          <h1 className={styles.header}>Select file(s) to upload</h1>
+          <h1 className={styles.header}>Select {uploadType}(s) to upload</h1>
           <div className={styles.dropZone} onClick={onBrowse}>
             <UploadOutlined className={styles.uploadIcon} />
             <div>Drag&nbsp;and&nbsp;Drop</div>
-            <div className={styles.dropZoneHelpText}>{helpText}</div>
+            <div className={styles.dropZoneHelpText}>or click to browse file paths</div>
```

Comment:
would "or click to browse files" be better for users here?

Reply:
Huh. Yeah, I think I like that better.

Reply:
Changed in d13ba59b39c7bea811fa978bbd7f425242f12f2e